webservices/resources/elections.py

```diff
@@ -115,15 +115,22 @@ def _filter_zip(self, query, kwargs):
     description=docs.ELECTIONS,
     tags=['financial']
 )
-class ElectionView(utils.Resource):
+class ElectionView(ApiResource):
+    schema = schemas.ElectionSchema
+    page_schema = schemas.ElectionPageSchema
+    @property
+    def args(self):
+        return utils.extend(
+            args.paging,
+            args.elections,
+            args.make_sort_args(
+                default='-total_receipts',
+            ),
+        )
 
-    @use_kwargs(args.paging)
-    @use_kwargs(args.elections)
-    @use_kwargs(args.make_sort_args(default='-total_receipts'))
-    @marshal_with(schemas.ElectionPageSchema())
-    def get(self, **kwargs):
+    def build_query(self, *args, **kwargs):
         query = self._get_records(kwargs)
-        return utils.fetch_page(query, kwargs, cap=0)
+        return query
 
     def _get_records(self, kwargs):
```

Comment:
Could we just rename this to `build_query` with the right signature?

Reply:
Yes, good point. will do. Thanks